[RUN-8862] addedd to RN and files (#352)
run-ai/docs: docs/Researcher/user-interface/workspaces/create/create-compute.md

To create a compute resource:

* Select the `New Compute Resource` button.

> * In the *Scope* pane, choose one or more items from the tree. The compute resource is assigned to one or more projects in the scope.

Reviewer: ```suggestion
* In the *Scope* pane, choose one item from the tree. The compute resource is assigned to that item and all its subsidiaries.
```
